src/AutoRest.CSharp/Common/Output/Expressions/Samples/Snippets.ExampleValues.cs

```diff
@@ -188,13 +190,35 @@ private static ValueExpression GetExpressionForFrameworkType(Type frameworkType,
                 return Null.CastTo(frameworkType);
             }
 
-            return frameworkType.IsValueType ? Default.CastTo(frameworkType) : Null.CastTo(frameworkType);
+            // TODO -- this might be an issue. We have so many shared common types in resoucemanager which replace the generated types in a library.
```

Comment:
No - others might not be fine.
I handled these two because I saw issues in the generated code for them.
We could add more when we see more in samples generated in azure-sdk-for-net